tests/test_rag_comprehensive.py

```
@@ -0,0 +1,675 @@
+        # Check actual keys returned by get_stats
+        assert "health" in result or "collection" in result or "cache" in result
+
+
+class TestRagServiceMCPIntegration:
+    """Test RAG service integration through MCP tools."""
+
+    @pytest.mark.asyncio
+    async def test_rag_query_tool(self, test_server):
+        """Test rag_query MCP tool."""
+        # First add some content using scrape tool
+        scrape_result = await test_server.call_tool(
```

Ruff (0.12.2)

Local variable `scrape_result` is assigned to but never used

Remove assignment to unused variable `scrape_result`

(F841)